core/src/main/java/org/ehcache/core/spi/cache/Store.java

```diff
@@ -181,20 +221,27 @@
    *       return true;
    *   } else return false;</pre>
    * except that the action is performed atomically.
+   * </P>
+   * <P>
+   * Neither the key nor the value can be {@code null}.
+   * </P>
    *
    * @param key key with which the specified value is associated
    * @param oldValue value expected to be associated with the specified key
    * @param newValue value to be associated with the specified key
    * @return {@link ReplaceStatus} based on the result of the replace operation in store
+   *
    * @throws ClassCastException if the specified key or values are not of the correct types ({@code K} or {@code V})
-   * @throws NullPointerException if the specified key or value is null
+   * @throws NullPointerException if any of the arguments is {@code null}
    * @throws CacheAccessException if the mapping can't be replaced
    */
   ReplaceStatus replace(K key, V oldValue, V newValue) throws CacheAccessException;
 
   /**
-   * Removes all of the mappings from this map.
-   * This method provides no guarantee in terms of atomicity.
+   * Removes all of the mappings from this {@code Store}.
+   * <P>
+   * This method provides no guarantee in terms of atomicity in the face of concurrent operations.
```

Comment:
I think "in terms" is redundant here. "This method provides no guarantee of atomicity in the face of concurrent operations" - although maybe even the last half is unnecessary too.